src/ui/src/app/dashboard/dashboard-body/main/attendance-module/daily-attendence/daily-attendence.component.ts

```diff
@@ -65,12 +66,29 @@ export class DailyAttendenceComponent {
   convertToTime(dateStr: string): string {
     if(dateStr)
     {
-      const date = new Date(dateStr); 
-      const hours = date.getHours().toString().padStart(2, '0'); 
-      const minutes = date.getMinutes().toString().padStart(2, '0'); 
-      return `${hours}:${minutes}`; 
+      const split=dateStr.split('+')[0];
+      const DateRemove=split.split('T')[1];
+      return `${DateRemove}`
+    }
+    return 'N/A' 
+  }
+  convertToDate(dateStr: string)
+  {
+    if(dateStr)
+      {
+        const split=dateStr.split('+')[0];
+        const DateRemove=split.split('T')[0];
+        return `${DateRemove}`
+      }
+      return 'N/A'
+  }
+  isMissed(missPunch:string)
+  {
+    if(missPunch)
+    {
+      return missPunch;

```

Comment:
missPunch ?? 'N/A'

Reply:
if he is not present or misspunch is empty,instead of that N/A will be seen .Modified to '-' if no miss punch